README.md

```diff
@@ -108,30 +108,31 @@ hCaptcha.setup().verifyWithHCaptcha()
 1. The listeners (`onSuccess`, `onFailure`, `onOpen`) can be called multiple times in the following cases:
    1. the same client is used to invoke multiple verifications
    2. the config option `resetOnTimeout(true)` is used which will automatically trigger a new verification when the current token expired. This will result in a new success or error callback.
-
+   3. `onFailure` with `TOKEN_TIMEOUT` will be called once the token is expired. To prevent this you can call `HCaptchaTokenResponse.markUsed` once the token is utilized. Also, you can change expiration timeout with `HCaptchaConfigBuilder.tokenExpiration(timeout)` (default 2 min.)
```

Comment:
The following scenario could be a bit confusing:

1. hCaptcha client is set up
2. verification is triggered and token1 is issued. 2min timer to expire this token1 has started.
3. sleep 1 min
4. new verification is triggered and token2 is issued. 2min timer to expire this token2 has started.
5. sleep 1 min. Error `TOKEN_EXPIRED` is emitted.  hCaptcha client cannot identify which token is expired: token1 or token2.

Should we just remove the timers for all previous issued tokens for the current hCaptcha client when a new verification process is started? @CAMOBAP @e271828-

Reply:
Not a problem to implement this, but in this case, the first token will never expire + it doesn't look like obvious behavior

> hCaptcha client cannot identify which token is expired: token1 or token2.

He can because a user of our SDK (developer) can pass different OnFailureListener instances

P.S. no critical objections from my side

Reply:
I'd vote for consistency, as @DSergiu outlined above.

Reply:
Done